build.sbt

```diff
@@ -113,10 +114,7 @@ lazy val commonSettings = Seq(
   libraryDependencies += "org.typelevel" %%% "discipline-core" % disciplineV,
   libraryDependencies += {
     if (isDotty.value)
-      ("org.specs2" %%% "specs2-scalacheck" % specs2V)
-        .cross(CrossVersion.for3Use2_13)
-        .exclude("org.scalacheck", "scalacheck_2.13")
-        .exclude("org.scalacheck", "scalacheck_sjs1_2.13")
+      "org.specs2" %%% "specs2-scalacheck" % specs2DottyV
     else
       "org.specs2" %%% "specs2-scalacheck" % specs2V
```

Comment:
This makes me really nervous because it makes it very hard for users to easily cross-build between Scala 2 and Scala 3.

Reply:
@rossabaker suggested in https://github.com/typelevel/discipline-specs2/issues/192#issuecomment-907707311 that the next version of discipline-specs2 is Scala 3 only.

Reply:
That's probably reasonable. It'll be more of a problem for cats-effect-testing though